iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -329,16 +283,17 @@ static LogicalResult setX86SandboxRootConfig(
                      /*workgroupSize=*/ArrayRef<int64_t>{});
 
   // Hardcoded tile sizes. The configuration is derived from iree-llvm-sandbox.
-  // L1 tile sizes are {1, 1, ..., 288, 128, 512}.
-  // Vector tile sizes are {1, ..., 9, 32, 16}
+  // L1 tile sizes are {1, 1, ..., 288, 128, 512}. Note that L1 tiling sizes is
```

Comment:
Could we just not set them. Also not clear why the L1 tile sizes are unused. If set here they will be used. These should be set to the same sizes as the second level tile sizes of sandbox.

Reply:
Ok, I see wy they arent used. I'd rather just not set them, but upto you. Using the L1 tile sizes and not using the vector tile sizes is more intuitive to me.

Reply:
I think we might want it at some point. The first level of tiling is applied at flow level. It only considers parallel dims. In sandbox, the DoulbeTilingExpert tiles both parallel dims and reduction dims for first level of tiling. The TileFuseAndVectorize pass also does similar things. I think we'd like to keep L1 tiling size and use it soon. I sent the PR out because the performance looks fine to me. I can give it a shot and update the status in the PR later.

Reply:
There are some compilation issues when trying it. I'll go with using L1 tile sizes (not vector tile sizes) at this moment and drop a TODO here.